npm/react/plugins/utils/wrap-devserver.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function returnInjectDevServerFunction (devServer, postPocessConfig = () => {}) {
+  return (...args) => {
+    // Old CT plugin signature: setupDevServer(on, config)
+    if (typeof args[0] === 'function') {
+      const [on, config, additionalOptions] = args
+
+      on('dev-server:start', (options) => {
+        return devServer(options, additionalOptions)
+      })
+
+      return postPocessConfig(config)
+    }
+
+    // New CT plugin signature: setupDevServer(options)
+    const [options, additionalOptions] = args
+
+    return devServer(options, additionalOptions)
```

Comment:
If we're going to have a `postProcessConfig` we also need to call it with the `setupDevServer(options)` signature

Reply:
We cannot postProcess the config in this case, can we?

Reply:
as discussed with Jessica, I don't think this should be part of the PR

Reply:
Is there a followup issue to address this? I want to make sure these dev server modules behave consistently when used with the new signature. /cc @JessicaSachs